azure-pipelines.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+trigger:
+  branches:
+    include:
+      - '*'
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+container: 'maven:3.6-jdk-8-slim'
+
+steps:
+- task: Maven@3
+  displayName: 'mvn install'
+  inputs:
+    mavenPomFile: 'pom.xml'
+    goals: 'install'
+    publishJUnitResults: true
+    testResultsFiles: '**/surefire-reports/TEST-*.xml'
+    javaHomeOption: 'JDKVersion'
+    mavenVersionOption: 'Default'
+    mavenAuthenticateFeed: false
+    sonarQubeRunAnalysis: false
+    pmdRunAnalysis: true
+
+- task: Bash@3
+  displayName: 'test example project with gradle'
+  inputs:
+    targetType: 'inline'
+    script: 'rm -rf example-projects/example-project/target
+      cd example-projects/example-project
```

Comment:
You probably need to write the script as it would be in line e.g.
'rm -rf example-projects/example-project/target && \
      cd example-projects/example-project && \
      gradle analyze && \
      java -jar ../../core/frontend-stubs-testing/target/antenna-test-project-asserter.jar ExampleTestProject build'